### DISCUSSION TITLE
[Enhancement] [table] 建议将带冻结功能的表格组件采用单table实现，本人可提供原生HTML5+CSS实现示例

### DISCUSSION THREAD

**Discussion by airsoft007** [element-plus/element-plus#18754] [Feature Request]
## Description

Element UI的Table组件，为了实现表格在滚动时表头固定、左右侧冻结列固定，采用了多个table联动实现。当表体滚动时，需用程序控制表头、左右侧列联动滚动。当数据量略大时，页面将变得非常卡顿，性能降低。实际上，采用单table纯CSS即可实现该效果。本人可提供示例例程。因本贴无法上传附件，EL团队若有需要，可联系本人QQ：5655585。

### Additional comments

可联系本人QQ：5655585，提供实现源码示例。加Q时备注：冻结表格Demo

![121221](https://github.com/user-attachments/assets/a6a03888-c7ce-468c-b219-93c32e6ca1fc)

**Comment by HaceraI** [comment]
是通过 position: sticky; 来实现冻结列吗？
el-table 现在确实有性能问题。

**Comment by elarbor** [comment]
表格就该用一个组件，就叫`<el-table>` 。现在是两个[`<el-table>` ](https://element-plus.org/zh-CN/component/table.html)和[`<el-table-v2>`](https://element-plus.org/zh-CN/component/table-v2.html) 。

用type区分更好一点：
`<el-table type="default">` 就是原来的`<el-table>`组件
`<el-table type="virtualized">`  就是原来的`<el-table-v2>` 组件
`<el-table type="第三种">`  就是以后的`<第三种>` 组件。    ......
